src/main/resources/assets/sodium/lang/en_us.json

```diff
@@ -53,13 +55,19 @@
   "sodium.options.use_no_error_context.tooltip": "When enabled, the OpenGL context will be created with error checking disabled. This slightly improves rendering performance, but it can make debugging sudden unexplained crashes much harder.",
   "sodium.options.buttons.undo": "Undo",
   "sodium.options.buttons.apply": "Apply",
+  "sodium.options.buttons.close": "Close",
   "sodium.options.buttons.donate": "Buy us a coffee!",
+  "sodium.prompt.donate.contents": "Hello!\n\nIt seems that you've been enjoying §x27eb92Sodium§r, the free and open-source optimization mod for Minecraft.\n\nMods like these are complex. They require §xff6e00thousands of hours§r of development, debugging, and tuning to create the experience that players have come to expect.\n\nIf you'd like to show your token of appreciation, and support the development of our mod in the process, then consider §xed49cebuying us a coffee§r.\n\nAnd thanks again for using our mod! We hope it helps you (and your computer.)",
+  "sodium.prompt.donate.action": "Buy us a coffee",
   "sodium.console.game_restart": "The game must be restarted to apply one or more video settings!",
   "sodium.console.broken_nvidia_driver": "Your NVIDIA graphics drivers are out of date!\n  * This will cause severe performance issues and crashes when Sodium is installed.\n  * Please update your graphics drivers to the latest version (version 536.23 or newer.)",
   "sodium.console.pojav_launcher": "PojavLauncher is not supported when using Sodium.\n  * You are very likely to run into extreme performance issues, graphical bugs, and crashes.\n  * You will be on your own if you decide to continue -- we will not help you with any bugs or crashes!",
   "sodium.console.core_shaders_error": "The following resource packs are incompatible with Sodium:",
   "sodium.console.core_shaders_warn": "The following resource packs may be incompatible with Sodium:",
   "sodium.console.core_shaders_info": "Check the game log for detailed information.",
+  "sodium.console.config_corrupt.title": "Could not load the configuration file",
+  "sodium.console.config_corrupt.short": "Sodium failed to load the configuration file",
+  "sodium.console.config_corrupt.contents": "A problem occurred while trying to load the configuration file. This\ncan happen when the file has been corrupted on disk, or when trying\nto manually edit the file by hand.\n\nIf you continue, the configuration file will be reset back to known-good\ndefaults, and you will lose any changes that have since been made to your\\nVideo Settings.\n\nMore information about the error can be found in the log file.",
```

Comment:
Newlines don't work here
![image](https://github.com/CaffeineMC/sodium-fabric/assets/61277953/597ed040-922f-45a0-951e-ab9188f30260)

Reply:
Should be fixed

Reply:
![yes](https://github.com/CaffeineMC/sodium-fabric/assets/8611110/5bc0caf6-e4ac-4946-a315-180fd6d77338)